.circleci/config.yml

```diff
@@ -0,0 +1,67 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10.15.1
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}
+      - run:
+          name: Install things from npm
+          command: npm install
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}
+          paths:
+            - ./node_modules
+      - run:
+          name: Build the site
+          command: npm run build
+      - persist_to_workspace:
+          root: .
+          paths:
+            - build
+
+  deploy:
+    docker:
+      - image: olizilla/ipfs-dns-deploy
+        environment:
+          DOMAIN: explore.ipld.io
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - run:
+          name: Pin website, post notification for PRs or update DNS on master
+          command: |
+            hash=$(ipfs-cluster-ctl --host /dnsaddr/cluster.ipfs.io \
+                                    --basic-auth $CLUSTER_USER:$CLUSTER_PASSWORD \
+                                    add --rmin 3 --rmax 3 --name "$DOMAIN" \
+                                    --recursive /tmp/workspace/build | tail -n1 | cut -d " " -f 2 )
+            # "preload" hash on the gateway
+            curl https://ipfs.io/ipfs/$hash
+            echo "Website available at https://ipfs.io/ipfs/$hash"
+            if [ ! -z "$CIRCLE_PULL_REQUEST" ] ; then
+              # comment="Website preview: https://ipfs.io/ipfs/$hash"
+              # pr_num=$(echo ${CIRCLE_PULL_REQUEST##*/})
+              # comment_url=$(curl -H "Authorization: token $GITHUB_TOKEN" https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/pulls/$pr_num | jq -r ._links.comments.href)
+              # curl -H "Authorization: token $GITHUB_TOKEN" -X POST -d "{\"body\": \"$comment\"}" $comment_url
+              echo "TODO: github comment"
+            elif [ "$CIRCLE_BRANCH" == "master" ] ; then
+              # Update DNSLink for domain
+              dnslink-dnsimple -d $DOMAIN -r _dnslink -l /ipfs/$hash
```

Comment:
I grabbed the build of dnslink-dnsimple from https://github.com/ipfs/dnslink-dnsimple/pull/8#issuecomment-439595409 and added it to the the ipfs-dns-deploy docker image https://github.com/ipfs-shipyard/ipfs-dns-deploy

@scout can you eyeball this before I mash run it for realz

Reply:
the context env vars are working nicely btw!

Reply:
see https://circleci.com/gh/ipfs-shipyard/ipld-explorer/16 is pinning to cluster! huzzah!